instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -352,18 +382,39 @@ class OpenTelemetryMiddleware:
     """
 
     def __init__(
-        self, wsgi, request_hook=None, response_hook=None, tracer_provider=None
+        self,
+        wsgi,
+        request_hook=None,
+        response_hook=None,
+        tracer_provider=None,
+        meter_provider=None,
     ):
         self.wsgi = wsgi
         self.tracer = trace.get_tracer(__name__, __version__, tracer_provider)
+        self.meter = get_meter(__name__, __version__, meter_provider)
+        self.duration_recorder = self.meter.create_histogram(
+            "http.server.duration",
+            "ms",
+            "measures the duration of the inbound HTTP request",
+        )
+        self.active_requests_counter = self.meter.create_up_down_counter(
+            "http.server.active_requests",
+            "1",
```

Comment:
Should this be [`requests`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#http-server)?